feature-pack/src/main/resources/modules/system/layers/base/org/jboss/as/transactions/main/module.xml

```diff
@@ -58,11 +57,20 @@
         <module name="org.wildfly.common"/>
         <module name="org.wildfly.transaction.client"/>
         <module name="org.jboss.jandex"/>
-        <module name="org.jboss.remoting"/>
         <module name="org.jboss.modules"/>
         <module name="org.jboss.narayana.compensations"/>
+        <module name="org.wildfly.security.manager"/>
```

Comment:
My guess is because it's require regardless if the `org.wildfly.undertow.http-invoker` capability is present.

Reply:
@tomjenkinson I didn't see your question until just now.  I didn't really move o.w.security.manager. I moved io.undertow.core and o.w.http-client.transaction, which preceded it, and the diff presentation makes it look like a move of security.manager. I moved those other two to split them out a bit visually because the comment I added applies to both.

Reply:
Thanks Brian